docs/keplergl-jupyter/README.md

```diff
@@ -90,6 +90,16 @@ map_1
 
 ![empty map][empty_map]
 
+ By default, the User Guide URL (<https://docs.kepler.gl/docs/keplergl-jupyter>) will be printed when a map is created. To hide the User Guide URL, set `show_docs=False`.
```

Comment:
Move this to after `__`config`__` on `line 77`

Reply:
Updated. Thanks. 

![image](https://user-images.githubusercontent.com/5016453/127548154-c9995999-2fe2-4cb0-9676-9014c487ec43.png)